src/libs/dutil/thmutil.cpp

```diff
@@ -1078,6 +1083,21 @@ DAPI_(void) ThemeShowControl(
 }
 
 
+DAPI_(void) ThemeShowControlEx(
```

Comment:
Making this function have the same parameters as `ThemeShowControl` is weird.  Either it should have a more descriptive name or it should expose the extra functionality (like whether to save editboxes or whether it should be `DEFAULT` or `REFRESH`).